src/script/page/MainContent/panels/preferences/OptionPreferences.tsx

```diff
@@ -56,7 +56,7 @@ const OptionPreferences: React.FC<OptionPreferencesProps> = ({
   } = propertiesRepository;
   const [optionAudio, setOptionAudio] = useState<AudioPreference>(settings.sound.alerts);
   const [optionReplaceInlineEmoji, setOptionReplaceInlineEmoji] = useState<boolean>(settings.emoji.replace_inline);
-  const [optionDarkMode, setOptionDarkMode] = useState<boolean>(settings.interface.theme === ThemeViewModelThemes.DARK);
+  const [optionDarkMode, setOptionDarkMode] = useState<boolean>(settings.interface.theme === 'dark');
```

Comment:
This looks like a random string, but it's actually autocompleted and typechecked by typescript. No need for `const` for that anymore

Reply:
praise: I find literal strings even more readable than enums/consts (if they are strongly typed) 😎